contracts/upgradeable_contracts/amb_erc677_to_erc677/BasicAMBErc677ToErc677.sol

```diff
@@ -79,21 +88,29 @@ contract BasicAMBErc677ToErc677 is
         bridgeContract().requireToPassMessage(mediatorContractOnOtherSide(), data, requestGasLimit());
     }
 
-    function relayTokens(uint256 _value) external {
+    function relayTokens(address _from, address _receiver, uint256 _value) external {
```

Comment:
The same question as raised in https://github.com/poanetwork/poa-bridge-contracts/pull/294#discussion_r335434722 is applicable here.

Reply:
thanks